x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/predicate/regex/RegexOperation.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.core.expression.predicate.regex;
+
+import java.util.regex.Pattern;
+
+public class RegexOperation {
+
+    public static Boolean match(Object value, Pattern pattern) {
```

Comment:
I think this method is never used

Reply:
We can remove it in a follow-up; I'd like to keep this in place in this PR so this particular change is mechanical and easy to validate.